.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-yarn lint-staged
+yarn lint-staged --allow-empty
```

Comment:
why we need  `--allow-empty`

Reply:
It was for test .. I'm going to remove it